readme.txt

```diff
@@ -157,6 +157,25 @@ HTTPS is a prerequisite for progressive web apps. A service worker is only able
 
 At the moment the plugin provides an API to detection of whether a site supports HTTPS. Building on that it's intended that this can then be used to present a user with an opt-in to switch over to HTTPS, which will also then need to include support for rewriting URLs from HTTP to HTTPS. See [labeled GitHub issues](https://github.com/xwp/pwa-wp/issues?q=label%3Ahttps) and see WordPress core tracking ticket [#28521](https://core.trac.wordpress.org/ticket/28521).
 
+You can optionally add an [HSTS header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security) (HTTP `Strict-Transport-Security`). This indicates to the browser to only load the site with HTTPS, not HTTP.
+
+<pre lang=php>
+/**
+ * Adds an HSTS header to the response.
+ *
+ * @param array $headers The headers to filter.
+ */
+add_filter( 'wp_headers', function( $headers ) {
+	$headers['Strict-Transport-Security'] = 'max-age=3600'; // Or another max-age.
```

Comment:
This needs to return the `$headers` array.

Reply:
That's a big omission 😄 
https://github.com/xwp/pwa-wp/pull/100/commits/dd49a54cf4009a2b77a74809e4a3c349d57223ca adds the `return` statemetn.